plugins/russian/ranobelib.ts

```diff
@@ -132,10 +151,21 @@ class RLIB implements Plugin.PluginBase {
             "/c" + chapter.chapter_number +
             "?bid=" + (chapter.branch_id || ""),
           releaseTime: dayjs(chapter.chapter_created_at).format("LLL"),
-          chapterNumber: totalChapters - chapterIndex,
+          chapterNumber:
+            customOrder[chapter.branch_id || 0] - (chapter.index || 0) ||
+            totalChapters - chapterIndex,
+          page: customPage[chapter.branch_id || 0] || "Основной перевод",
         }),
       );
-      novel.chapters = chapters.reverse();
+      novel.chapters =
+        chaptersJson.chapters.branches?.length &&
+        chaptersJson.chapters.branches.length > 1
+          ? chapters.sort((a, b) => {
+              if ((a.page || 0) > (b.page || 0)) return 1;
+              if ((a.page || 0) < (b.page || 0)) return -1;
+              return (a.chapterNumber || 0) - (b.chapterNumber || 0);
+          })
+          : chapters.reverse();
```

Comment:
sorting by chapter number is enough. Sorting by page makes no sense if page is provided for chapters

Reply:
Without page sorting, the chapters in the appendix will alternate from one page to another as you read. 
But with page sorting, they move from one page to the next.